packages/mobile/src/import/ImportWallet.tsx

```diff
@@ -177,7 +177,7 @@ export class ImportWallet extends React.Component<Props, State> {
           </View>
         )}
         <GethAwareButton
-          disabled={isSubmitting || !this.state.backupPhrase}
+          disabled={isSubmitting || this.state.backupPhrase.trim().split(/\s+/g).length < 12}
```

Comment:
Consider creating an `isBackupPhraseValid` method and doing this check there

Reply:
Also, did you mean 24 instead of 12?

Reply:
The issue considers both: 12 and 24. Is it ok?

Reply:
> Consider creating an `isBackupPhraseValid` method and doing this check there

Sure!